docs-ref-autogen/azure-mgmt-apicenter/azure.mgmt.apicenter.ApiCenterMgmtClient.yml

```diff
@@ -8,7 +8,7 @@ inheritances:
 summary: Azure API Center Resource Provider.
 constructor:
   syntax: 'ApiCenterMgmtClient(credential: TokenCredential, subscription_id: str,
-    base_url: str = ''https://management.azure.com'', **kwargs: Any)'
+    base_url: str = ''`https://management.azure.com`'', **kwargs: Any)'
```

Comment:
`syntax` gets rendered to code blocks, we don't want anything modified in these.